0.2.0 release (#61)
port-labs/Port-Ocean: CHANGELOG.md

<!-- towncrier release notes start -->

0.2.0 (2023-08-09)
==================

### Breaking Changes

> - Updated the `on_resync` generator to use a list of items instaed of single item

Reviewer: ```suggestion
- Updated the `on_resync` generator to use a list of items instead of a single item
```

### Improvements

- Changed default log level to `INFO`
- Changed the extra object messages log level from `INFO` to `DEBUG`
- Removed a wrongful error log at the integration installation that says the integration does not exists

### Bug Fixes

> - Supporting many relations for the same entity (PORT-4379)

Reviewer: ```suggestion
- Added support for many relations for the same entity (PORT-4379)
```

> - Pass resource config inside event context (PORT-4398)

Reviewer: ```suggestion
- Added the resource config to the event context (PORT-4398)
```

> - Fix not supporting multiple relations (PORT-4411)

Reviewer: ```suggestion
- Fixed lack of support for multiple relations (PORT-4411)
```

> - Log exception traceback of integration resync methods exceptions (PORT-4422)

Reviewer: ```suggestion
- Added traceback output to the integration resync method exception log (PORT-4422)
```
